cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/AbstractSparkContext.java

```diff
@@ -256,15 +259,19 @@ public DiscoveryServiceClient getDiscoveryServiceClient() {
    * Returns the underlying {@link Metrics} instance for emitting user metrics. The returned instance is
    * not serializable and shouldn't be exposed to user program directly.
    */
-  public Metrics getUserMetrics() {
-    return userMetrics;
+  public Metrics getProgramWorkflowMetrics() {
```

Comment:
the method name was updated to be in accordance with the comment for this method.